File: 0929 - Unique Email Addresses/solution.py
class Solution:
    def numUniqueEmails(self, emails: List[str]) -> int:
        s = set()
        for i in emails:
            local, domain = i.split('@')
            local = local.split('+')[0].replace('.', '')
            s.add((local, domain))
        return len(s)

File: 0929 - Unique Email Addresses/solution.cpp
class Solution {
public:
    int numUniqueEmails(vector<string>& emails) {
        unordered_set<string> s;
        for(auto& i : emails){
            string local, domain = i.substr(i.find('@'));
            for(auto c : i){
                if(c=='+' || c=='@') break;
                if(c == '.') continue;
                local += c;
            }
            s.insert(local+domain);
        }
        return s.size();
    }
};